### ISSUE TITLE
Link the volume of the sample-pre-listening to the master volume

### ISSUE THREAD

**Issue by fpiesik** [xiphonics/picoTracker#75]
When master volume is turned down, the pre listening of samples is still loud, it seems not to be affected by the master volume.

**Comment by maks** [comment]
Yes currently the sample preview is done by streaming the sample straight from sdcard in  `AudioFileSreamer` class instead of the usual code path for import sample playback. 
In that class, there is code that reads a `config.xml` file value `PRELISTENATTENUATION`, but that's not one of the config variables that is supported in picoTracker so there is no way to set a value for it as that value won't be read out of the `config.xml` because its not in the white-list in `sources/Application/Model/Config.cpp`.
 
My first thought would be to either apply the existing value of the master volume as you mentioned here or perhaps have a separate volume in the `PRELISTENATTENUATION` value as the code is now but also add a key combo to adjust that volume value in the import sample screen, as I think you would want to adjust there and have that saved into config.xml to persist between power cycles, versus just setting it via editing `config.xml` which is not convenient on the picoTarcker.

**Comment by maks** [comment]
In the case of using the separate sample preview volume value `PRELISTENATTENUATION` we would also need to copy across the xml saving code to `sources/Application/Model/Config.cpp` as currently it only supports reading not writing.

Also it should be noted that setting output volume in general is currently an open issue: #13 

Not sure what you think about this @democloid ?

**Comment by fpiesik** [comment]
I believe that applying the value of the master volume to PRELISTENATTENUATION would be the most elegant and time-efficient approach. I can't think of many use cases where the volume of the pre-listening should differ from the overall volume. Besides, there are so many other cool things to do :)

**Comment by democloid** [comment]
It's a completely different codepath, so it seems things are going to be kept in sync continuously, which is annoying. I think it makes sense to match this to master output volume, but if hard to do for now, I think we should probably just attenuate it enough so that it makes sense for most use cases as a fixed value.

**Comment by maks** [comment]
I think it should be doable to use master volume, I'll try to tackle this soon.

**Comment by maks** [comment]
Fixes in #365
